platooning_strategic_IHP/src/platoon_strategic_ihp.cpp

```diff
@@ -1322,7 +1860,8 @@ namespace platoon_strategic_ihp
 
     // UCLA: add conditions to account for frontal join states (candidate follower) 
     void PlatoonStrategicIHPPlugin::mob_resp_cb_leader(const cav_msgs::MobilityResponse& msg)
-    {   /**  
+    {   
+        /**  
```

Comment:
I like this description of the functionality. But what if the host is the current platoon leader?  What mob_op_req would this be responding to, and what would the msg.plan_type be?  My guess is that all 3 of the bools below would be false, but it would be nice to see a comment confirming that.